conan/api/subapi/cache.py

```diff
@@ -45,6 +46,13 @@ def package_path(self, pref: PkgReference):
         ref_layout = app.cache.pkg_layout(pref)
         return ref_layout.package()
 
+    @api_method
+    def check_integrity(self, package_list):
```

Comment:
This has been moved from the ``UploadAPI`` to here, because checking package integrity in the cache is not exclusively a "upload" operation. This will enable the ``conan cache check-integrity`` command with just a couple of lines.